tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/media_storage.py

```diff
@@ -91,12 +62,72 @@ def confirm_select_media_location(self):
         Wait(self.marionette).until(expected.element_displayed(*self._default_change_ok_button_locator))
 
     def pick_media_location(self,location):
-        if location is 'Internal':
+        if location == 'Internal':
             _selection_locator = (By.CSS_SELECTOR, '[data-option-index="0"]')
-        elif location is 'SD Card':
+        elif location == 'SD Card':
             _selection_locator = (By.CSS_SELECTOR, '[data-option-index="1"]')
-        self.marionette.find_element(*_selection_locator).tap()
+        else:
+            raise AttributeError('{} is not a media supported in the test'.format(location))
+        element = Wait(self.marionette).until(expected.element_present(*_selection_locator))
+        Wait(self.marionette).until(expected.element_displayed(element))
+        element.tap()
         self.marionette.find_element(*self._default_change_ok_button_locator).tap()
         self.apps.switch_to_displayed_app()
         Wait(self.marionette).until(expected.element_displayed(*self._media_location_selector_locator))
 
+    class StorageRegion(PageRegion):
+        _music_size_locator = (By.CSS_SELECTOR, '[data-l10n-id="music-space"] + [data-l10n-id="storageSize"]')
+        _movies_size_locator = (By.CSS_SELECTOR, '[data-l10n-id="videos-space"] + [data-l10n-id="storageSize"]')
+        _pictures_size_locator = (By.CSS_SELECTOR, '[data-l10n-id="pictures-space"] + [data-l10n-id="storageSize"]')
+        _format_locator = (By.CSS_SELECTOR, '[data-l10n-id^="format-sdcard"]')
+        _eject_locator = (By.CSS_SELECTOR, '[data-l10n-id^="eject-sdcard"]')
+
+        _format_dialog_cancel_locator = (By.ID, 'format-sdcard-cancel-btn')
+        _format_dialog_confirm_locator = (By.ID, 'format-sdcard-ok-btn')
+        _eject_dialog_cancel_locator = (By.ID, 'unmount-sdcard-cancel-btn')
+        _eject_dialog_confirm_locator = (By.ID, 'unmount-sdcard-ok-btn')
+
+        @property
+        def music_size(self):
+            return self.root_element.find_element(*self._music_size_locator).text
+
+        @property
+        def movies_size(self):
+            return self.root_element.find_element(*self._movies_size_locator).text
+
+        @property
+        def pictures_size(self):
+            return self.root_element.find_element(*self._pictures_size_locator).text
+
+        @property
+        def format_button(self):
```

Comment:
I see why you defined the button as a property, that's a good thing. However, I wouldn't expose it as public. Let's rename it to `_format_button`, so people will know that's a protected attribute